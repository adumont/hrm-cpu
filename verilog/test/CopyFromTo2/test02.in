03 ed
19 c8
09 58
30 76
23 d2
50 cc
48 95
05 30
32 df
32 10
03 b1
09 51
31 06
04 30
53 87
12 7b
27 c8
31 ba
18 5f
12 5b
13 4e
22 d7
08 ef
19 a7
58 94
32 6b
50 73
55 73
02 a6
59 b1
52 37
05 ac
58 22
29 a0
51 f3
19 f3
21 ef
01 f9
47 16
41 18
19 d9
48 17
22 26
