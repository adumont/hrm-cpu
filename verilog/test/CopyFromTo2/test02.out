ed
c8
58
76
d2
cc
95
30
df
10
b1
51
06
30
87
7b
c8
ba
5f
5b
4e
d7
ef
a7
94
6b
73
73
a6
b1
37
ac
22
a0
f3
f3
ef
f9
16
18
